src/ucp/api/device/ucp_device_types.h

```diff
@@ -73,7 +73,7 @@ typedef struct ucp_device_mem_list_handle {
     /**
      * Array of lengths of the local buffers in bytes.
      */
-    size_t *lengths;
+    size_t   *lengths;
```

Comment:
was the intent to align with other firlds?
Otherwise why need this change?

Reply:
Yes. Maybe I will do it in a different PR because we have a big mess...
In `ucp_device_mem_list_params_t` it's `size_t`.
In `ucp_device_mem_list_handle_t` it's `uint32_t`.
In `put_multi/put_multi_partial` it's `unsigned`.